mcbackend/core.py

```diff
@@ -152,11 +152,12 @@ def constant_data(self) -> Dict[str, numpy.ndarray]:
     def observed_data(self) -> Dict[str, numpy.ndarray]:
         return {dv.name: ndarray_to_numpy(dv.value) for dv in self.meta.data if dv.is_observed}
 
-    def to_inferencedata(self, **kwargs) -> InferenceData:
+    def to_inferencedata(self, equalize_chain_lengths: bool=True, **kwargs) -> InferenceData:
         """Creates an ArviZ ``InferenceData`` object from this run.
 
         Parameters
         ----------
+        equalize_chain_lengths: Whether to truncate all chains to the minimum chain length
```

Comment:
```suggestion
        equalize_chain_lengths : bool
            Whether to truncate all chains to the shortest chain length (default: ``True``).
```
just standardizing to the usual numpydoc style